src/dagmc/DagMC.hpp

```diff
@@ -571,8 +570,8 @@ inline EntityHandle DagMC::entity_by_index(int dimension, int index) const {
 }
 
 inline int DagMC::index_by_handle(EntityHandle handle) const {
-  assert(handle - setOffset < entIndices.size());
-  return entIndices[handle - setOffset];
+  assert(entIndices.count(handle) > 0);
+  return entIndices.at(handle);
```

Comment:
Is there a reason to use this instead of the `[]` operator?

Reply:
This method is `const`. The `[]` operator violates this as it would also support assignment.

Reply:
Thanks for the education!